packages/faustwp-cli/utils/shouldFireTelemetryEvent.ts

```diff
@@ -1,11 +1,17 @@
 import { userConfig } from './userConfig.js';
+import { getWpSecret } from '../../faustwp-core/src/lib/getWpSecret.js';
 
 export function shouldFireTelemetryEvent(): boolean {
-  const hasSecretKey = process.env.FAUSTWP_SECRET_KEY;
+  const args = getCliArgs();
+  const arg1 = args[0];
```

Comment:
```suggestion
```

Looks like this is a remnant of a bad merge. We can remove this.

Reply:
@josephfusco added these a month ago. Do we no longer need these? They are referenced a few lines below.